Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java

```diff
@@ -527,112 +526,114 @@ public static Uni<Tuple2<Object, List<Tuple2<String, String>>>> generateCompacte
 		Uni<Tuple3<String, String, List<Tuple2<String, String>>>> uni;
 		switch (acceptHeader) {
 
-		case 1:
-			uni = ldService.compact(entity, contextHeader, context, opts, -1, optionSet, langQuery).onItem()
-					.transformToUni(compacted -> {
-						List<Tuple2<String, String>> headers = Lists.newArrayList();
-						Object bodyContext = compacted.remove(NGSIConstants.JSON_LD_CONTEXT);
-						Object finalCompacted;
-						if (contextHeader.isEmpty()) {
-							contextHeader.add(((List<Object>) bodyContext).get(0));
-						}
-						if (compacted.containsKey(JsonLdConsts.GRAPH)) {
-							finalCompacted = compacted.get(JsonLdConsts.GRAPH);
-						} else {
-							finalCompacted = compacted;
-						}
-						if (options != null && options.contains(NGSIConstants.QUERY_PARAMETER_CONCISE_VALUE)) {
-							makeConcise(finalCompacted);
-						}
-						if (forceArray && !(finalCompacted instanceof List)) {
-							finalCompacted = List.of(finalCompacted);
-						}
+			case 1:
+				uni = ldService.compact(entity, contextHeader, context, opts, -1, optionSet, langQuery).onItem()
+						.transformToUni(compacted -> {
+							List<Tuple2<String, String>> headers = Lists.newArrayList();
+							Object bodyContext = compacted.remove(NGSIConstants.JSON_LD_CONTEXT);
+							Object finalCompacted;
+							if (contextHeader.isEmpty()) {
+								contextHeader.add(((List<Object>) bodyContext).get(0));
+							}
+							if (compacted.containsKey(JsonLdConsts.GRAPH)) {
+								finalCompacted = compacted.get(JsonLdConsts.GRAPH);
+							} else {
+								finalCompacted = compacted;
+							}
+							if (options != null && options.contains(NGSIConstants.QUERY_PARAMETER_CONCISE_VALUE)) {
```

Comment:
Please verify concise and normalized functionality.

Reply:
done